Description: Fix make clean target
Author: Raphael Hertzog <buxy@kali.org>
Origin: vendor
Last-Update: 2015-11-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -15,7 +15,7 @@ CFLAGS		+= -g3 -ggdb
 #CFLAGS		+= -static
 PROGOBJ		= md5.o sha1.o utils.o cowpatty.o genpmk.o
 PROG		= cowpatty genpmk
-BINDIR		= /usr/local/bin
+BINDIR		= /usr/bin
 
 all: $(PROGOBJ) $(PROG)
 
@@ -36,7 +36,7 @@ sha1: sha1.c sha1.h
 	$(CC) $(CFLAGS) sha1.c -c
 
 clean:
-	@rm $(PROGOBJ) $(PROG)
+	@rm -f $(PROGOBJ) $(PROG)
 
 strip:
 	@ls -l $(PROG)
